tests/token/token.js

```diff
@@ -26,7 +26,9 @@ describe( 'Token', () => {
 
 	describe( 'constructor()', () => {
 		it( 'should throw error when no tokenUrl provided', () => {
-			expect( () => new Token() ).to.throw( '`tokenUrl` must be provided' );
+			expect( () => new Token() ).to.throw(
+				'A `tokenUrl` or a `tokenRefresh` function must be provided as the first constructor argument.'
```

Comment:
It is interesting that this test is passing, because the message is about "refreshToken", not "tokenRefresh".

Reply:
I'll check it. Maybe the `to.throw()` doesn't test the string?

Reply:
I think, that it'd be good to close https://github.com/ckeditor/ckeditor-cloud-services-core/issues/18 in that PR too. Maybe it's connected.